### DISCUSSION TITLE
Contributions

### DISCUSSION THREAD

**Discussion by platima** [geerlingguy/sbc-reviews#30] [General]
Hey mate I've got a pile more SBCs that you don't have on the list - want me to run these benchmarks for you?

Off the top of my head, some of the others I've got which run some Linux kernel;
- Milk-V Mars (non-CM)
- MangoPi MQPro
- Milk-V Duo
- Raxda Rock 4 SE
- Pine64 A64 LTS
- Pine64 Rock64
- Luckfox Pico Plus, Pro and Max
- Pine64 Ox64
- BananaPi M1

I'd rather contribute to a central repo of this data than just start my own! Cheers

**Comment by geerlingguy** [comment]
Definitely! Any that you want to test, please feel free to create a new issue (the template should have all the tests and instructions... otherwise I'd forget 'em!).

But please don't feel obligated to do everything, do the ones you want to, and we'll slowly fill in blanks as time goes on :)

**Comment by platima** [comment]
Yeah no worries - it won't be something I can do on the spot, but as I 
use each one for something I'll flash a fresh image and run them.

So far it's rather heartbreaking how far behind the RISC-V processors 
are, I was really hoping the JH7110 would do better. Fingers crossed for 
the SG2380 but I feel like they're going quantity over quality there 😂


------ Original Message ------
From "Jeff Geerling" ***@***.***>
To "geerlingguy/sbc-reviews" ***@***.***>
Cc "Platima" ***@***.***>; "Author" ***@***.***>
Date 26/11/2023 8:00:38 AM
Subject Re: [geerlingguy/sbc-reviews] Contributions (Discussion #30)


Definitely! Any that you want to test, please feel free to create a new 
issue (the template should have all the tests and instructions... 
otherwise I'd forget 'em!).

But please don't feel obligated to do everything, do the ones you want 
to, and we'll slowly fill in blanks as time goes on :)

—
Reply to this email directly, view it on GitHub 
<https://github.com/geerlingguy/sbc-reviews/discussions/30#discussioncomment-7668931>, 
or unsubscribe 
<https://github.com/notifications/unsubscribe-auth/ADIYAQECI7FUVZ36E34GCCTYGKBCNAVCNFSM6AAAAAA72NMOEOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TMNRYHEZTC>.
You are receiving this because you authored the thread.Message ID: 
***@***.***>
